### ISSUE TITLE
BOTs not attacking enemy BOTs sometimes.

### ISSUE THREAD

**Issue by SmartShots** [Joe91/fun-bots#34]
I'm not sure on the exact requirements for this to happen yet but I can say that sometimes the BOTs are definitely not attacking enemy BOTs.

I think it may be to do with how the BOTs were spawned but I need to test this.

**Comment by Joe91** [comment]
maybe it was related to some wierd teamId behaviour on levelstart. Can you check if it is still happening in the new master?

**Comment by SmartShots** [comment]
Going off what was reported in issue #28 - I have tested by having 5 BOTs spawned by default and then adding more BOTs from the in game UI. Initially I was testing with master build 1.6.0.1 and all tests were done on the Noshahr Canals TDM map with spawnMode = 'increment_with_players' and onlySpawnBotsWithPlayers = true.

With 5 BOTs spawned by default and the server filled by adding BOTs via the UI - The BOTs added don't seem to be attacking the enemy BOTs and the initial BOTs are rarely attacking.

With 30 BOTs spawned by default - The BOTs are attacking the enemy BOTs.

I then updated to master build 1.6.1.2.

With 5 BOTs spawned by default and the server filled by adding BOTs via the UI - All the BOTs are now attacking the enemy BOTs.

With 30 BOTs spawned by default - The BOTs are attacking the enemy BOTs.

Looks like the new master fixed it. This can be closed now.
